Core/include/Acts/Geometry/CompositePortalLink.hpp

```diff
@@ -0,0 +1,45 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Geometry/Portal.hpp"
+#include "Acts/Geometry/PortalLinkBase.hpp"
+
+#include <boost/container/small_vector.hpp>
+
+namespace Acts {
+
+class CompositePortalLink final : public PortalLinkBase {
+ public:
+  void toStream(std::ostream& os) const final { os << "CompositePortalLink"; }
```

Comment:
should this be moved to source?

also are we missing includes / forwards for `std::ostream`?

Reply:
I'll move it. Indeed there's no explicit include / forward. I can add it.